From fb0d8074778af9f12b0e84472d509eac2aa52dd0 Mon Sep 17 00:00:00 2001
From: Brian Behlendorf <behlendorf1@llnl.gov>
Date: Thu, 4 Feb 2021 09:56:28 -0800
Subject: zfs-list.8: clarify listing snapshots

Clarify how to include snapshots in the `zpool list` output by
referencing the full name of the `listsnapshots` pool property,
and the `zpool list -t snapshot` option.

Reviewed-by: Matthew Ahrens <mahrens@delphix.com>
Reviewed-by: George Melikov <mail@gmelikov.ru>
Signed-off-by: Brian Behlendorf <behlendorf1@llnl.gov>
Closes #11562
Closes #11565
---
 man/man8/zfs-list.8 | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/man/man8/zfs-list.8 b/man/man8/zfs-list.8
index e6db73631..3ed74955e 100644
--- a/man/man8/zfs-list.8
+++ b/man/man8/zfs-list.8
@@ -63,12 +63,17 @@ If specified, you can list property information by the absolute pathname or the
 relative pathname.
 By default, all file systems and volumes are displayed.
 Snapshots are displayed if the
-.Sy listsnaps
-property is
+.Sy listsnapshots
+pool property is
 .Sy on
 .Po the default is
 .Sy off
-.Pc .
+.Pc ,
+or if the
+.Fl t Sy snapshot
+or
+.Fl t Sy all
+options are specified.
 The following fields are displayed:
 .Sy name Ns \&, Sy used Ns \&, Sy available Ns \&, Sy referenced Ns \&, Sy mountpoint Ns .
 .Bl -tag -width "-H"
-- 
2.30.0

